ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/RunUntilFailureTask.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+using YamlDotNet.Serialization;
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks
+{
+    public class RunUntilFailureTask : GithubTask
```

Comment:
Suggest a name change so it is clear that it is referring to dotnet test runner

```suggestion
    public class RunDotNetTestUntilFailureTask : GithubTask
```